### DISCUSSION TITLE
Responsible Design Playbook (beyond SustainableUX)

### DISCUSSION THREAD

**Discussion by SandrineResponsibleDesigner** [Green-Software-Foundation/discussions#90] [Standards Working Group]
Hi there!

Me and my team at Method are working on a **Responsible Design Playbook**.
That's updating our design approach, process and Design Ops to provide a guide to truly creating digital services responsibly (good for people and planet).
@adamj89 @NAMRATA-WOKE : Is there a working group on this topic? 
Any fellow Experience Designers, Strategists, Product Designers, Product Managers, Makers in the network willing to join the effort in establishing new standards? 

<img width="616" alt="Screenshot 2023-11-17 at 09 41 24" src="https://github.com/Green-Software-Foundation/discussions/assets/150163340/445bfc84-7ceb-4e5b-801f-2a4494cc0c28">

<img width="516" alt="Screenshot 2023-11-17 at 09 47 42" src="https://github.com/Green-Software-Foundation/discussions/assets/150163340/7661be90-005d-493e-bbae-c8a7a6e1b38a">

<img width="517" alt="Screenshot 2023-11-17 at 09 46 49" src="https://github.com/Green-Software-Foundation/discussions/assets/150163340/32399cdb-d93e-4ae8-8555-578ecf02c69c">

**Comment by russelltrow** [comment]
Hi @SandrineResponsibleDesigner,

I was really excited to read about your work on a responsible design playbook, I had previously seen your Design Card Library.

I'm a GSF PM under the Community Working Group focused on our Training, Patterns & Maturity Matrix projects. In terms of overlap between your efforts and the GSF, our [Green Software Patterns project](https://patterns.greensoftware.foundation/) could be the most relevant, especially as we are starting to explore how we can expand the patterns into other areas such as Product & Design.

If you would like to drop me an email on russell[at]greensoftware.foundation I would be more than happy to provide more of an introduction.

Russ

**Reply by NAMRATA-WOKE** [reply]
Thanks for chiming in @russelltrow. @SandrineResponsibleDesigner I'm happy to find time to connect to share more about the foundation, especially if Method is interested in becoming a member!

**Reply by SandrineResponsibleDesigner** [reply]
Hi @russelltrow 
(sorry emerging out of a delivery tunnel) – would be great to be able to contribute to the GSF Pattern project.
Contacting you by email to discuss this further.

@NAMRATA-WOKE I'll connect with you separately as well – thanks for connecting :)
